Tests/SecureXPCTests/Client & Server/Server Termination Integration Test.swift

```diff
@@ -19,52 +19,45 @@ class ServerTerminationIntegrationTest: XCTestCase {
     }()
     
     // This test is intended to simulate a server running in a different process and then that process terminating
-    func testShutdownServer() throws {
+    func testShutdownServer() async throws {
         // Server & client setup
-        let echoRoute = XPCRoute.named("echo").withMessageType(String.self).withReplyType(String.self)
+        let route = XPCRoute.named("doNothing")
         let server = XPCServer.makeAnonymous(clientRequirements: dummyRequirements)
-        try server.registerRoute(echoRoute) { msg in
-            return "echo: \(msg)"
-        }
+        try server.registerRoute(route) { }
         server.start()
         let client = XPCClient.forEndpoint(server.endpoint)
         
         // Send a message, which will result in the connection being established with the server
-        client.sendMessage("1st message", route: echoRoute) { _ in }
+        try await client.send(route: route)
         
         // Shut down the server, simulating the scenario of the process containing the server terminating
         (server as! XPCAnonymousServer).simulateDisconnectionForTesting()
         
-        // Make two more calls after having shutdown the server:
-        // - The second call should fail upon trying to send the message, connection is interrupted
-        // - The third call (and any subsequent ones) should fail indicating no new connections can ever be established
         let interruptedExpectation = self.expectation(description: "Second message results in an interrupted error")
-        let cannotBeReestablishedExpectation = self.expectation(description: "Third message can't be sent")
-        client.sendMessage("2nd message", route: echoRoute) { response in
-            switch response {
-                case .failure(.connectionInterrupted):
+        do {
+            try await client.send(route: route)
```

Comment:
Ah, looks like [`XCTAssertThrowsError(_:_:file:line:_:)`](https://developer.apple.com/documentation/xctest/1500795-xctassertthrowserror) can't take an `async` expression, eh?

Reply:
Yeah. As I just learned, making `async` versions of everything is rather a pain. I figure it's going to be a multiyear journey until everything that ought to be or support Swift concurrency does. Certainly complicated by the fact that Objective-C doesn't have a direct equivalent and it's going to be important for large established code bases for many years to come.

Reply:
Looks like [Quick](https://github.com/Quick/Quick) doesn't support it [yet](https://github.com/Quick/Quick/issues/1084), either.

I have larges for Quick to become the standard testing framework in the Swift ecosystem. It happens to use XCTest under the hood, but pretty much entirely decouples you from it, to isolate you from the madness (e.g. `XCAssertEqual` makes no indication as to which param should be "expected" vs "actual", and a ton of codebases (including mine) use it wrong/inconsistently T_T)